queries/ecma/injections.scm

```diff
@@ -1,5 +1,5 @@
 (comment) @jsdoc
```

Comment:
Doesn't this also apply to us jsdoc? Are really both parsers required?

Reply:
Think it was decided to use both, jsdoc and comment for the injection. Don't know if jsdoc should be combined.

Reply:
jsdocs has the same performance problem on my machine

Reply:
Changed